client/src/redux/reducers/userReducer.js

```diff
@@ -1,17 +1,17 @@
 import { createReducer } from '@reduxjs/toolkit';
-import { setUser, setIsExistUser } from '../actions/userAction';
+import { setUser, setIsAuthed } from '../actions/userAction';
 
 
 const initialState = {
   user: {},
```

Comment:
Можна зробити ще компактніше, і використовувати лише поле user.
Щось на приклад такого
```suggestion
  user: sessionStorage.getItem('token'),
```
а потім в перевірках робити не isAuthed, а просто !!user

Reply:
Тоді поле isAuthed можна видалити